src/library_dylink.js

```diff
@@ -256,7 +266,7 @@ var LibraryDylink = {
   $dlSetError: function(msg) {
     withStackSave(function() {
       var cmsg = allocateUTF8OnStack(msg);
-      ___dl_seterr(cmsg);
+      ___dl_seterr(cmsg, 0);
```

Comment:
why?

Reply:
___dl_seterr is a vararg function (like printf).  It takes a C string pointer and pointer to varargs.  With wasm32 we can just not specify the second argument and the VM with create a i32 of zero in its place.  With wasm64 we are forced to pass the second argument... because __dl_seterr is marked as `_pp` in  `create_wasm64_wrappers` the conversion of both cmsg and 0 to BigInt is don't automatically by the wrappers.